source/content/professional-services.md

```diff
@@ -25,21 +27,11 @@ The Site Migration service lowers the barrier to entry by leveraging our PS team
 
 ## Managed Updates
 
-Pantheon's [Managed Updates](/managed-updates) service keeps your site up to date with minimal interruption to end users. Allow Pantheon’s PS team to apply core and contributed plugin/module updates.
```

Comment:
This should be updated for the link to the new page.

Reply:
even with the link at the end of that section? I wasn't sure about doubling it up like that

Reply:
:derp: - as you were.